tdrs-backend/tdpservice/data_files/models.py

```diff
@@ -132,3 +176,111 @@ def find_latest_version(self, year, quarter, section, stt):
         return self.objects.filter(
             version=version, year=year, quarter=quarter, section=section, stt=stt,
         ).first()
+
+class LegacyFileTransferManager(models.Manager):
```

Comment:
> Could someone clarify to me why this is called legacy?

Because we will be transferring these to the legacy TDRS system. That's all this SFTP epic is for.